src/components/index.ts

```diff
@@ -0,0 +1,64 @@
+import dynamic from 'next/dynamic'
+
+const Layout = dynamic(async () => await import('@/components/layout/Layout'), {
+  ssr: false
+})
+
+const PhaseCard = dynamic(async () => await import('@/components/card/phase/PhaseCard'), {
+  ssr: false
+})
+
+const BaseCard = dynamic(async () => await import('@/components/card/base/BaseCard'), {
+  ssr: false
+})
+
+const Accordion = dynamic(async () => await import('@/components/accordion/Accordion'), {
+  ssr: false
+})
+
+const Podium = dynamic(async () => await import('@/components/podium/Podium'), {
+  ssr: false
+})
+
+const LeaderboardTable = dynamic(async () => await import('@/components/table/LeaderboardTable'), {
+  ssr: false
```

Comment:
I have a question: what have you disabled the SSR for the whole components?

Reply:
Sorry, completely forgot to remove this.